### ISSUE TITLE
Rule to detect agpl-3-0-plus_or_commercial actually triggers on AGPL-3.0-plus without commercial remarks

### ISSUE THREAD

**Issue by zesoup** [nexB/scancode-toolkit#3740]
### Description

The current rule [agpl-3.0-plus_or_commercial-license_4.RUL](https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/rules/agpl-3.0-plus_or_commercial-license_4.RULE) classifies AGPL short descriptions as (AGPL-plus or commercial).

However, while the usecase may often be linked to commercial licenses the exact snippet within the rule does not mention commercial availability at all hence i suggest changing this to omit the "or commercial" part of things.

**Comment by pombredanne** [comment]
@zesoup good catch! this is a mistake and a bug. This rule https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/rules/agpl-3.0-plus_or_commercial-license_4.RULE should be strictly about AGPL and has nothing about commercial!

**Comment by AyanSinhaMahapatra** [comment]
This was requalified with https://github.com/aboutcode-org/scancode-toolkit/commit/d76977d6d414a7a3ea48d3539674dea20607fb3e to just `agpl-3.0-plus` and now lives at https://github.com/aboutcode-org/scancode-toolkit/blob/develop/src/licensedcode/data/rules/agpl-3.0-plus_299.RULE, old rule is deprecated. Thanks @zesoup, closing!
